tests/integration/test_project.py

```diff
@@ -42,6 +45,43 @@ def test_project(client, rand_gen):
     assert project not in projects
 
 
+def test_batch_project_export_v2(
+        configured_batch_project_with_label: Tuple[Project, Dataset, DataRow,
```

Comment:
Nit: We don't need to define all the return types since this is a test. May be cleaner to not have to import all of the types too